edge-modules/Relayer/Program.cs

```diff
@@ -62,6 +66,9 @@ static async Task<MessageResponse> ProcessAndSendMessageAsync(Message message, o
                     messageCopy.Properties.Add(prop.Key, prop.Value);
                 }
 
+                TestResultCoordinatorClient trcClient = new TestResultCoordinatorClient { BaseUrl = testResultCoordinatorUrl.AbsoluteUri };
```

Comment:
I think trcClient should be created once before this method which gets called for each received message